InOut/CMakeLists.txt

```diff
@@ -83,13 +83,12 @@ if(APPLE)
         target_link_libraries(rtauhal PRIVATE ${AUDIOUNIT_LIBRARY} ${COREFOUNDATION_LIBRARY} ${COREAUDIO_LIBRARY})
 endif()
 
-if(USE_ALSA AND LINUX)
-    if (USE_VCPKG)
-        find_package(ALSA MODULE REQUIRED)
-    else()
-        find_package(ALSA REQUIRED)
-    endif()
+if (NOT TARGET ALSA::ALSA)
```

Comment:
`find_package(ALSA)` seems to only work once, and it might have already been called

Reply:
Where is the code that finds Alsa?

Reply:
It's a build-in cmake module: https://cmake.org/cmake/help/latest/module/FindALSA.html